cmd/frontend/build_frontend_env/build-frontend-env-file.go

```diff
@@ -13,6 +13,7 @@ import (
     "github.com/spf13/cobra"
     "github.com/DSGT-DLP/Deep-Learning-Playground/cli/cmd/frontend" // For frontend/
     "github.com/DSGT-DLP/Deep-Learning-Playground/cli/utils" // For utils/
+    "encoding/json" // to unmarshal json
```

Comment:
i dont think you need the // to unmarshall json comment. its a bit redundant bc your implementation explains this